media-gfx/openvdb/openvdb-11.0.0.ebuild

```diff
@@ -175,21 +175,21 @@ src_prepare() {
 }
 
 my_src_configure() {
-	local version
+	local version abi_version
 	version=$(ver_cut 1)
 	if use "abi$(( version + 1 ))-compat"; then
```

Comment:
Your changes don't affect when `use` is called, which implies the error from the bug can still happen:
```
 [31;01m*[0m ERROR: media-gfx/openvdb-11.0.0::gentoo failed (configure phase):
 [31;01m*[0m   USE Flag 'abi8-compat' not in IUSE for media-gfx/openvdb-11.0.0
 [31;01m*[0m 
```

You could use `in_iuse ... && use ....` although that may mean you have dead logic sitting around.

Reply:
The error was in https://github.com/negril/gentoo/blob/media-gfx/openvdb/11.0.0_fix_abi/media-gfx/openvdb/openvdb-11.0.0.ebuild#L225 because `version` was modified for `!abi11-compat`. Now only `abi_version` is modified and used for `-DOPENVDB_ABI_VERSION_NUMBER="${abi_version}"`.